docs/cado-response/deploy/aws/iam/cross-cloud.md

```diff
@@ -36,7 +36,7 @@ Cross-cloud access of GCP from AWS is performed by designating a primary GCP pro
 * Create a new [GCS bucket](https://cloud.google.com/storage/docs/creating-buckets) for storing evidence collected by Cado. This bucket will be selected during the import process in the Cado Response interface. 
 * Create a new Cado IAM role with the [permissions listed here](https://docs.cadosecurity.com/cado-response/deploy/gcp/gcp-settings#creating-a-cado-role.).
 This role can be [created](https://cloud.google.com/iam/docs/creating-custom-roles#creating) at the organisation level to give access to multiple projects under one service account.
-* Assign the new Cado IAM role to the Cloud Build principal. Go to IAM and Admin -> IAM and locate the principle which looks like xxxxxxxxxxxx@cloudbuild.gserviceaccount.com. Edit this principle to give it permission to the new Cado role. Take note of the principal name as it will be used to set up access to other GCP projects.
+* Assign the new Cado IAM role to the Compute Engine principal - note that as of June 2024 the Compute Engine principal needs to be updated instead of the Cloud Build prinicpal due [to changes in GCP](https://cloud.google.com/build/docs/cloud-build-service-account-updates). Go to IAM and Admin -> IAM and locate the principle which looks like xxxxxxxxxxxx@cloudbuild.gserviceaccount.com. Edit this principle to give it permission to the new Cado role. Take note of the principal name as it will be used to set up access to other GCP projects.
```

Comment:
The service account for Compute will differ from the cloud build one, so the example here should be changed to:
xxxxxxxxxxxx-compute@developer.gserviceaccount.com

The same change will be needed on the 'adding-aws-gcp.md' page